script/uninstall.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+set -eu
+
+# Uninstalls Zed that was installed using the install.sh script
+# Modeled after install.sh
+
+main() {
+    platform="$(uname -s)"
+    channel="${ZED_CHANNEL:-stable}"
+
+    if [ "$platform" = "Darwin" ]; then
+        platform="macos"
+    elif [ "$platform" = "Linux" ]; then
+        platform="linux"
+    else
+        echo "Unsupported platform $platform"
```

Comment:
Let's support FreeBSD here also, as it had been recently semi-supported in https://github.com/zed-industries/zed/pull/20480

Reply:
The installation script doesn't support freebsd and I'm not ready to take any responsibility for it working correctly